src/js/media-time-range.js

```diff
@@ -148,6 +152,17 @@ class MediaTimeRange extends MediaChromeRange {
   //   // TODO: Reset value after media is unset
   // }
 
+  get mediaDuration() {
+    return +this.getAttribute(MediaUIAttributes.MEDIA_DURATION);
+  }
+
+  get mediaBuffered() {
+    const buffered = this.getAttribute(MediaUIAttributes.MEDIA_BUFFERED);
+    if (buffered) {
+      return buffered.split(' ').map((timePair) => timePair.split(','));
```

Comment:
non-blocking: should this be moved to a function alongside `serializeTimeRanges` in some utils file? `parseTimeRangeString`? `deserializeTimeRanges`?

Reply:
until it's needed a 2nd time maybe, with this pure vanilla ESM approach I'd prefer also not to make more file / dependency requests than necessary.